### ISSUE TITLE
Fonts not loaded with webpack 2

### ISSUE THREAD

**Issue by larsbs** [webpack/webpack#4462]
**Do you want to request a *feature* or report a *bug*?**

Report a bug.

**What is the current behavior?**

When I use any `css` or `less` code that contains urls to fonts assets, those assets are not loaded. When the same `css` code is used in a link tag, the fonts are correctly loaded.

![screen shot 2017-03-10 at 13 53 38](https://cloud.githubusercontent.com/assets/1436588/23796070/1ad77958-0599-11e7-8706-ff92611e224d.png)

In this screenshot, `dbi.css` is the `css` required through a link tag and `icons.woff` is the file it's requiring. `8ada1f3f-b867-46ed-ac7e-3385f0d5874f` is the blob required by webpack, as can be seen, no extra request to fetch the fonts it's made.

In the following screenshots, it can be seen that apart from minification, the code it's the same.

![screen shot 2017-03-10 at 13 53 59](https://cloud.githubusercontent.com/assets/1436588/23796109/68aee6d4-0599-11e7-8b71-9a2d39af25af.png)

![screen shot 2017-03-10 at 13 53 50](https://cloud.githubusercontent.com/assets/1436588/23796101/580a02a0-0599-11e7-877b-ad4adab2b53e.png)

**If the current behavior is a bug, please provide the steps to reproduce.**

Simply clone this [repo](https://github.com/larsbs/webpack-fonts-issue) and run `yarn start`, then visit `http://localhost:4001/` and open dev tools network inspector.

**What is the expected behavior?**

Fonts should be required as in any normal css file.

**If this is a feature request, what is motivation or use case for changing the behavior?**

**Please mention other relevant information such as the browser version, Node.js version, webpack version and Operating System.**

**Comment by timse** [comment]
why should the browser issue a request if it already knows the font?

**Comment by larsbs** [comment]
Comment out the `link` tag in the project. Then the browser still not request the font even if it's a font it doesn't know.

<img width="731" alt="screen shot 2017-03-11 at 14 53 44" src="https://cloud.githubusercontent.com/assets/1436588/23823797/99891280-066a-11e7-87cb-c42886952fca.png">

<img width="731" alt="screen shot 2017-03-11 at 14 53 51" src="https://cloud.githubusercontent.com/assets/1436588/23823799/9e47460c-066a-11e7-9f8e-11a25473ffe0.png">

**Comment by larsbs** [comment]
Any news on this? is it a bug? is it happening to more people? is a problem with my configuration?

**Comment by timse** [comment]
I can reproduce as in there is no request going out, but I wouldnt say its necessarily a webpack issue. (webpack is not responsible for making network requests).

**Comment by sokra** [comment]
see style-loader: https://github.com/webpack-contrib/style-loader#converttoabsoluteurls

**Comment by larsbs** [comment]
I have tried with that option, and is still not working. I've updated the sample repo to reflect that. And also, I know that webpack is not responsible for making network request, but if it's a behavior that was working in webpack 1, it should be the same in webpack 2 or mentioned as a caveat in the documentation. I've created a branch called `webpack1` in the sample repo running the same code with version 1 of webpack that works correctly.
